src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -196,30 +262,74 @@ const TableToolbar = ({
           disabled={isDisabled}
         />
       ) : null}
-      {customToolbarContent || null}
-      {totalFilters > 0 ? (
-        <Button kind="secondary" onClick={onClearAllFilters}>
-          {i18n.clearAllFilters}
-        </Button>
-      ) : null}
-      {onDownloadCSV ? (
-        <ToolbarSVGWrapper onClick={onDownloadCSV}>
-          <Download16 description={i18n.downloadIconDescription} />
-        </ToolbarSVGWrapper>
-      ) : null}
-      {hasColumnSelection ? (
-        <ToolbarSVGWrapper onClick={onToggleColumnSelection}>
-          <IconColumnSelector description={i18n.columnSelectionButtonAria} />
-        </ToolbarSVGWrapper>
-      ) : null}
-      {hasFilter ? (
-        <ToolbarSVGWrapper onClick={onToggleFilter}>
-          <IconFilter description={i18n.filterButtonAria} />
-        </ToolbarSVGWrapper>
-      ) : null}
-    </StyledTableToolbarContent>
-  </StyledCarbonTableToolbar>
-);
+      {activeEditBar === 'buttons' ? (
+        <StyledTableToolbarContent>
+          <Button className="buttons" kind="ghost" onClick={onCancelEditAction}>
+            {i18n.batchCancel}
+          </Button>
+          <Button className="buttons" onClick={onSaveEditAction}>
+            {i18n.batchSave}
+          </Button>
+        </StyledTableToolbarContent>
+      ) : activeEditBar === 'toast' ? (
+        <ToastNotification
+          className="toast"
+          caption=""
+          hideCloseButton={false}
+          iconDescription="undo changes and close"
```

Comment:
Not sure where the toast stuff is coming from.  Is there something in the spect that says that the table is manage the showing and hiding of the toast??  This seems completely unneccary.  Not to mention that all the descriptions and labels  now need to be translated (which they are hard coded strings here).  If everyone agrees... let's just move showing a toast as part of the story.

Reply:
I'm here to listen to everyone's opinions😄. I wrote the toast stuff entirely based on the design.

Reply:
I agree that it seems weird to have a toast as part of this component. The popup placement of the toast also seems visually off. @lukefirth @amymillett any comments on this from a visual/UX perspective? I second Sean's recommendation to just show the toast as part of the story to reinforce the pattern but not make a part of the Table component.